pkg/metricsDashboard/logger.go

```diff
@@ -103,3 +104,14 @@ func TrackDeal(params DealPayload) {
 	url := host + namespace + "/" + dealsEndpoint
 	http.GenericJSONPostClient(url, payload)
 }
+
+func TrackHashrate(hashrate data.MinerHashRate) {
```

Comment:
Please move this function to the [`powLogs` package](https://github.com/Lilypad-Tech/lilypad/blob/main/pkg/powLogs/logger.go)